openqa-trigger-bisect-jobs

```diff
@@ -109,11 +109,14 @@ client_args = [
 
 
 def call(cmds, dry_run=False):
-    log.debug("call: %s" % cmds)
-    return subprocess.check_output(
-        (["echo", "Simulating: "] if dry_run else []) + cmds
-    ).decode("utf-8")
-
+    try:
```

Comment:
This is now making every system call non-fatal.
We still want unsuccessful system calls aborting the script, just without the long stack trace.
Also returning the stderr output might now trigger other exceptions for code that expects the system call to return JSON.